### ISSUE TITLE
Add killed pod metric

### ISSUE THREAD

**Issue by mblaschke** [linki/chaoskube#150]
It would be nice to have a metric which pod has been killed with labels like `namespace` and `pod` (name of pod).

So teams can have a easy monitoring for their killed pods.

**Comment by linki** [comment]
@mblaschke Do you have a proposal how this metric could look like in prometheus format?

**Comment by zbindenren** [comment]
I imagine something like this:

```
# HELP chaoskube_terminated_pods_total Total number of terminated pods by namespace
# TYPE chaoskube_terminated_pods_total counter
chaoskube_terminated_pods_total{pod_namespace="namespaces"} 1
```

I am not sure if it is a good idea to add the podname, this could lead to an increased label [cardianality](https://www.robustperception.io/cardinality-is-key).

**Comment by mblaschke** [comment]
would be fine for me

**Comment by linki** [comment]
There's already a metric that counts the total amount of terminated pods. I added the namespace as a label there in https://github.com/linki/chaoskube/pull/341.

@zbindenren @mblaschke Let me know if that works for you.

**Comment by zbindenren** [comment]
@linki I had also a PR ready :smile: 

Thanks, that is perfekt.

**Comment by linki** [comment]
@zbindenren Thank you 😃 

For reference, you can now use the following metric with the namespace label:
```
chaoskube_pods_deleted_total{namespace="baz"} 1
```

This should be available from version `v0.22.0` onwards.
